rest/handler.go

```diff
@@ -572,6 +580,11 @@ func (h *handler) validateAndWriteHeaders(method handlerMethod, accessPermission
 			keyspaceScope = base.StringPtr(base.DefaultScope)
 			keyspaceCollection = base.StringPtr(base.DefaultCollection)
 		}
+		if explicitCollectionLogging {
```

Comment:
I think a comment here as well describing why/when we're choosing explicit/implicit CollectionLogging would be helpful.